features/spaces.feature

```diff
@@ -12,4 +12,10 @@ Feature: Spaces
   Scenario: Space Entrances
     Given a Space with a Room specified as its Entrance
     When Anyone visits the Space
-    Then they see the Furniture in the Entrance Room
\ No newline at end of file
+    Then they see the Furniture in the Entrance Room
+
+  @built @unimplemented-steps
+  Scenario: Space without Entrances
+    Given a Space with no Entrance
+    When Anyone visits the Space
+    Then they only see the Rooms
```

Comment:
TO Perhaps we could unpack this into two assertions:

1. `Then they do not see any Furniture`  and 
2. `And they see the Room Picker for the Space` (which may already exist?)